strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/artifact/MockedMavenArtifactCoordinates.java

```diff
@@ -0,0 +1,186 @@
+package org.carlspring.strongbox.artifact;
+
+import org.carlspring.strongbox.artifact.coordinates.AbstractArtifactCoordinates;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.Map;
+
+import org.apache.maven.artifact.versioning.ComparableVersion;
+
+/**
+ * @author carlspring
+ */
+@XmlRootElement(name = "maven-artifact-coordinates")
+@XmlAccessorType(XmlAccessType.NONE)
+public class MockedMavenArtifactCoordinates
```

Comment:
This "useless" class was wherever you moved it from, because it's sometimes required to test something with "dummy" coordinates in modules which are generic and have no actual implementation of `ArtifactCoordinates` exposed to them via a dependency, as it would cause a circular dependency.

It's not great, but it does the job for testing purposes. I would suggest restoring it to it's original location, as the need for something like this occurs every now and then.

Reply:
will revert it back to the previous state :)

Reply:
Thanks! :)